pyproject.toml

```diff
@@ -22,7 +22,12 @@ classifiers = [
     "License :: OSI Approved :: MIT License",
     "Natural Language :: English",
     "Programming Language :: Python",
-    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
```

Comment:
```suggestion
```
Life's too short to keep this up to date. Plus it's redundant with `requires-python`.

Reply:
Yeah, agreed. I used https://github.com/tox-dev/pyproject-fmt to format this a bit and it seems to add those by default :\

Reply:
Removed in https://github.com/inducer/pytools/compare/1d13af56d6045ff02b087795cb4963f5c4c04623..a46be3f13216dd9781758fb96e91b0a3917e1154